### PR TITLE
Add TiCDC glossary and update upgrade-tidb-using-tiup

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

+ Add TiCDC glossary
+ Add more information in upgrade-tidb-using-tiup.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5471 https://github.com/pingcap/docs-cn/pull/5312
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by ran-huang** [comment]
Is there a Chinese PR?

**Comment by TomShawn** [comment]
> Is there a Chinese PR?

Oh, sorry. https://github.com/pingcap/docs-cn/pull/5471 https://github.com/pingcap/docs-cn/pull/5312

File: ticdc/ticdc-glossary.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Owner
+
+A [capture](#capture) of a special role that manages the TiCDC cluster and schedules replication tasks of the cluster. An owner is elected by captures and there is at most one owner at any time.
+
+## P
+
+### Processor
+
+TiCDC replication tasks allocate data tables on TiCDC instances and the processor refers to the replication processing unit of these tables. Processor tasks include pulling, sorting, restoring, and distributing changed data.
```
Comment (ran-huang) [inline ticdc/ticdc-glossary.md:34]:
```suggestion
TiCDC replication tasks allocate data tables on TiCDC instances, and the processor refers to the replication processing unit of these tables. Processor tasks include pulling, sorting, restoring, and distributing changed data.
```

**Review by ran-huang — APPROVED** [review]

File: ticdc/ticdc-glossary.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: TiCDC Glossary
+summary: Learn the terms about TiCDC and their definitions.
+---
+
+# TiCDC Glossary
+
+This glossary provides TiCDC-related terms and definitions. These terms appears in TiCDC logs, monitoring metrics, configurations, and documents.
+
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply ticdc/ticdc-glossary.md:9]: Sure.
```suggestion

For TiDB-related terms and definitions, refer to [TiDB glossary](/glossary.md).

```

File: ticdc/ticdc-glossary.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+For TiDB-related terms and definitions, refer to [TiDB glossary](/glossary.md).
```
Comment (TomShawn) [inline ticdc/ticdc-glossary.md:10]: Follow-up: need to add this line in docs-cn.

**Pull request merged**